test/functional/cmdLineTests/shareClassTests/SCCMLTests/exclude_openj9.xml

```diff
@@ -263,4 +263,64 @@
 	<exclude id="Test 65-b: Make sure none of the classes has been stored in the cache" platform="8">
 		<reason>Only Java 11 and up use jdk.internal.misc.Unsafe  </reason>
 	</exclude>
+	<exclude id="Test 180: Setup for Tests from 181 - 196. Create the layer 0 cache" platform="win_x86-32">
```

Comment:
Needs to be excluded for other non-Windows 32/31 bit platforms. You can try using something similar to: 
https://github.com/eclipse/openj9/blob/master/test/functional/cmdLineTests/shareClassTests/SCCMLTests/ShareClassesCMLTests-1.xml#L62